module/platform/darwin/HardwareIO/src/Convert.cpp

```diff
@@ -85,15 +85,15 @@ namespace platform {
         float Convert::servoPosition(const uint8_t id, const uint16_t value) {
             // offset and normalize the angle
             return utility::math::angle::normalizeAngle(
-                (((value - 2048) * POSITION_CONVERSION_FACTOR) + SERVO_OFFSET[id]) * SERVO_DIRECTION[id]);
+                (((value - 2048) * POSITION_CONVERSION_FACTOR) * SERVO_DIRECTION[id]) + SERVO_OFFSET[id]);
```

Comment:
It will but I already fixed it in the configuration files. Still wouldn't hurt to run in sensors test before running for real